lib/rubocop/cop/lint/unified_integer.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for using Fixnum or Bignum constant.
+      #
+      # @example
+      #   # bad
+      #   1.is_a?(Fixnum)
+      #   1.is_a?(Bignum)
+      #
+      #   # good
+      #   1.is_a?(Integer)
+      class UnifiedInteger < Cop
```

Comment:
The semantics slightly change starting in 2.4 so it probably makes most sense to only enable for 2.4 https://blog.blockscore.com/new-features-in-ruby-2-4/#simplified-integers